server/services/matchChat.ts

```diff
@@ -1,5 +1,5 @@
 import { IMatchMessage, IResponseProps } from '../interfaces';
-import { Match, Message } from '../models';
+import { Match, Message, Player, Stadium, User } from '../models';
 
 const addMessageService = async ({
```

Comment:
on get and add messages
in case the match is done or does not exist what will happen

Reply:
How can a message to be without sender for it

Reply:
in case someone sends a postman request:!